src/query.rs

```diff
@@ -164,6 +168,9 @@ impl SemverQuery {
     }
 }
 
+// TODO: replace with LazyLock once MSRV is >= 1.80
+pub(crate) static ALL_QUERIES: OnceLock<BTreeMap<String, SemverQuery>> = OnceLock::new();
```

Comment:
I really dislike globals like this. They make testing a mess.

Can you think of any reasonable way we can avoid this?